controllers/suite_test.go

```diff
@@ -206,8 +205,6 @@ func TestMain(m *testing.M) {
 		panic(fmt.Sprintf("Failed to create a test registry server: %v", err))
 	}
 
-	fg := feathelper.FeatureGates{}
-	fg.SupportedFeatures(features.FeatureGates())
```

Comment:
We still want to keep supported features here, as this tells to the flag parser what is allowed as option and we still support `features.OptimizedGitClones`.

Reply:
The flag parser is just for the CLI iirc. The reconciler has it's own features map (ref: https://github.com/fluxcd/source-controller/blob/main/controllers/gitrepository_controller.go#L121), which is configured by calling `features.FeatureGates()` below.